node/actors/bft/src/testonly/ut_harness.rs

```diff
@@ -83,7 +83,7 @@ impl UTHarness {
     /// Creates a new `UTHarness` with minimally-significant validator set size.
     pub(crate) async fn new_many(ctx: &ctx::Ctx) -> (UTHarness, BlockStoreRunner) {
         let num_validators = 6;
-        assert!(validator::faulty_replicas(num_validators) > 0);
+        assert!(validator::max_faulty_weight(num_validators as u64) > 0);
```

Comment:
just assert max_fault_weight on the `UTHarness::new().replica.validators` instead

Reply:
But that will fail on a number of existing tests that are creating validator committees with [one](https://github.com/matter-labs/era-consensus/blob/bf36b3c2844f742a83a51e553f49c86b11faa78a/node/actors/bft/src/leader/tests.rs#L381) or [two](https://github.com/matter-labs/era-consensus/blob/bf36b3c2844f742a83a51e553f49c86b11faa78a/node/actors/bft/src/leader/tests.rs#L478) validators only

Reply:
you should put the assert in the new_many constructor only. Just like it is now.

Reply:
```
let (x,y) = UTHarness::new(Ctx, num_validators).await;
assert!(x.genesis().validators.max_faulty_weight()>0);
(x,y)
```